### PR TITLE
*: remove backup side car

### PR DESCRIPTION
**Pull request by fanminshi**
ref:https://github.com/coreos/etcd-operator/issues/1626

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

File: hack/build/operator/build [hunk 1]
```diff
@@ -40,7 +40,6 @@ gitHash="github.com/coreos/etcd-operator/version.GitSHA=${GIT_SHA}"
 go_ldflags="-X ${ldKVPairs} -X ${gitHash}"
```
Comment (hongchaodeng) [inline hack/build/operator/build]: Change this script to only do build. Now it also does push.

Comment (fanminshi) [reply hack/build/operator/build]: @hongchaodeng do you want me to do the above in this pr or a separate pr?

Comment (hongchaodeng) [reply hack/build/operator/build]: Do it in separate pr.

**Comment by hongchaodeng** [comment]
lgtm

**Pull request merged**